config/splurt/autocryo.txt

```diff
@@ -0,0 +1,5 @@
+## Uncomment to disable automatic SSD cryo
+# AUTOCRYO_DISABLED
```

Comment:
Would be better if instead of a flag to disable it there was a flag to _enable_ it and that it came disabled by default